src/ir/expr.cc

```diff
@@ -38,6 +38,8 @@ PrimExpr::PrimExpr(int32_t value) : PrimExpr(IntImm(DataType::Int(32), value)) {
 
 PrimExpr::PrimExpr(float value) : PrimExpr(FloatImm(DataType::Float(32), value)) {}
 
+PrimExpr::PrimExpr(double value) : PrimExpr(FloatImm(DataType::Float(64), value)) {}
```

Comment:
This is dangerous and can cause problems in the codebase, since implicit efault conversion to double may not be intended from the user side, we would rather be explicit. Let us remove this constructor and work around it(by explicitly construct via make_const)